index.html

```diff
@@ -1,10 +1,55 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
 <head>
 	<meta charset="UTF-8">
 	<title>Title</title>
+	<link rel="stylesheet" href="css/fonts.css">
+	<link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+	<header>
+		<div class="header__technomart">
+			<div class="header__container">
```

Comment:
I think it will be the same container for all website, so you can use .container class that will be common for all cases

Reply:
Or other website use another container width for content?